src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -261,30 +283,47 @@ public static function setMapSetting(int $mapId, string $key, $value): void
      */
     public static function update(array $item): int
     {
-        // redefine edited on date
-        $item['edited_on'] = BackendModel::getUTCDate();
-
-        // we have an extra_id
-        if (isset($item['extra_id'])) {
-            // update extra
-            BackendModel::updateExtra(
-                $item['extra_id'],
-                'data',
-                [
-                    'id' => $item['id'],
-                    'extra_label' => \SpoonFilter::ucfirst(BL::lbl('Location', 'Core')) . ': ' . $item['title'],
-                    'language' => $item['language'],
-                    'edit_url' => BackendModel::createUrlForAction('Edit') . '&id=' . $item['id'],
-                ]
-            );
+        /** @var EntityRepository $repository */
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')->getRepository(Location::class);
```

Comment:
Why don't you just create an alias for this?
`$repository = BackendModel::get('locations.repository.location');`

And in `Backend/Modules/Locations/Resources/config/repositories.yml` add:
```yaml
services:
    locations.repository.location:
        class: Backend\Modules\Locations\Domain\Location\ Location Repository
        factory: ["@doctrine.orm.entity_manager", getRepository]
        arguments:
            - Backend\Modules\Locations\Domain\Location\Location
```